app/src/logging/formatters.py

```diff
@@ -11,53 +11,64 @@
 from datetime import date, datetime
 from decimal import Decimal
 from enum import Enum
-from typing import Any
+from typing import Any, Callable, Type
 from uuid import UUID
 
 import src.logging.decodelog as decodelog
 
 
+# identity returns unmodified value
+def identity(obj: Any) -> Any:
```

Comment:
what i'm about to say also doesn't matter at all to me, but if you _wanted_ to be more rigorous with typing you'd make this a generic, since the type of the return is the same as the parameter type, so it'd be:

```
from typing import TypeVar
T = TypeVar("T")
def identity(obj: T) -> T:
    return obj
```